pkgs/development/haskell-modules/default.nix

```diff
@@ -70,7 +70,8 @@ let
       };
 
     in
-      import ./hackage-packages.nix { inherit pkgs stdenv callPackage; } self // {
+      # TODO(@Ericson2314) think harder about build-time dependencies
+      import ./hackage-packages.nix { inherit stdenv callPackage; pkgs = pkgs.buildPackages // pkgs;  } self // {
```

Comment:
Stop-gap until cabal2nix learns buidlPackages and cross compiling. It uses gcc so needs wrappers which aren't in plain `pkgs` anymore.